Java/tests/src/test/java/com/microsoft/recognizers/text/tests/datetime/DateTimeParserTest.java

```diff
@@ -247,6 +260,35 @@ private static IDateTimeParser getSpanishParser(String name) {
                 throw new AssumptionViolatedException("Parser Type/Name not supported.");
         }
     }
+    private static IDateTimeParser getFrenchParser(String name) {
+
+        switch (name) {
+            case "DateParser":
+                return new BaseDateParser(new FrenchDateParserConfiguration(new FrenchCommonDateTimeParserConfiguration(DateTimeOptions.None)));
+            case "DatePeriodParser":
+                return new BaseDatePeriodParser(new FrenchDatePeriodParserConfiguration(new FrenchCommonDateTimeParserConfiguration(DateTimeOptions.None)));
+            //case "DateTimeAltParser":
```

Comment:
This is OK, as AlternativeTypes currently only exists for English and requires special flags to enable it.

Reply:
This is interesting. If I enable it, all tests still remain passing. Should I do so?